<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_command_9</class>
 <widget class="QWidget" name="form_command_9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Команда № 9 «Исключение КСЛ из группы»</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>149</number>
        </property>
        <property name="rightMargin">
         <number>48</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>123456</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>109</number>
        </property>
        <property name="rightMargin">
         <number>48</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Номер СОРМ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>68</number>
        </property>
        <property name="rightMargin">
         <number>48</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Номер группы КСЛ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>48</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Номер цифрового потока А</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>108</number>
        </property>
        <property name="rightMargin">
         <number>48</number>
        </property>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Номер КСЛ-А</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_10"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="rightMargin">
         <number>48</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Номер цифрового потока В</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_9"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="spacing">
         <number>108</number>
        </property>
        <property name="rightMargin">
         <number>48</number>
        </property>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Номер КСЛ-В</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_11"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Hex↴</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="text">
         <string>11 11 11 11 11 11 11 11 11 11 11 11 11</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>120</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Передать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
